Gopkg.toml

```diff
@@ -55,8 +55,7 @@ required = [
 
 [[override]]
   name = "k8s.io/client-go"
-  # revision for tag "kubernetes-1.10.0"
-  revision = "23781f4d6632d88e869066eaebb743857aa1ef9b"
+  version = "kubernetes-1.9.3"
```

Comment:
this version of k8s isn't compatible with service-catalog client. That's why I updated it to 1.10.0. To get it work with the operator-sdk, you can update all the k8s.io libs to 1.10.0. That's what I am going to do in my branch anway.

Reply:
will try this thanks